### DISCUSSION TITLE
Is ZMK tap hold the equivalent of press hold in VIA?

### DISCUSSION THREAD

**Discussion by worldtraveler1** [nickcoutsos/keymap-editor#172] [Q&A]
Hello. I new to ZMK having previously only used VIA/VIAL. I'm trying to implement the press-hold function for capslock when held acts as the option key on the mac. From reading [zmk.dev](https://zmk.dev/) I think Hold-Tap (&mt) is the quivalent function, however, I am unable to select CAPSLOCK as the tap while I can select option key as the hold. Does anyone know what I am doing wrong? Thanks.

[My current keymap](https://imgur.com/a/SmIh0vT)

**Comment by nickcoutsos** [comment]
It looks like you do have `CAPS` set as the tap for that binding so I'm not sure what your concern is. Also, this looks like a fork of keymap-editor, you might have better luck asking for help from its maintainer as this app has changed significantly since then.

**Comment by worldtraveler1** [comment]
The problem is that capslock doesn't work. It does nothing. The option key works though.

**Reply by nickcoutsos** [reply]
The app you're using isn't the app I'm writing. If you paid someone for a keyboard and they told you the keymap could be updated with their app then you will need their help.
